pkg/app/master/command/images/model.go

```diff
@@ -0,0 +1,166 @@
+package images
+
+import (
+	"time"
+
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/dustin/go-humanize"
+	"github.com/mintoolkit/mint/pkg/app"
+
+	"github.com/mintoolkit/mint/pkg/app/master/command"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/common"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/keys"
+	"github.com/mintoolkit/mint/pkg/crt"
+	"github.com/mintoolkit/mint/pkg/crt/docker/dockerutil"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Model represents the state of the TUI.
+type Model struct {
+	table      table.Table
+	width      int
+	height     int
+	standalone bool
+	loading    bool
+}
+
+// Styles - move to `common`
+const (
+	gray      = lipgloss.Color("#737373")
+	lightGray = lipgloss.Color("#d3d3d3")
+	white     = lipgloss.Color("#ffffff")
+)
+
+var (
+	// HeaderStyle is the lipgloss style used for the table headers.
+	HeaderStyle = lipgloss.NewStyle().Foreground(white).Bold(true).Align(lipgloss.Center)
+	// CellStyle is the base lipgloss style used for the table rows.
+	CellStyle = lipgloss.NewStyle().Padding(0, 1).Width(14)
+	// OddRowStyle is the lipgloss style used for odd-numbered table rows.
+	OddRowStyle = CellStyle.Foreground(gray)
+	// EvenRowStyle is the lipgloss style used for even-numbered table rows.
+	EvenRowStyle = CellStyle.Foreground(lightGray)
+	// BorderStyle is the lipgloss style used for the table border.
+	BorderStyle = lipgloss.NewStyle().Foreground(white)
+)
+
+// End styles
+
+func LoadModel() *Model {
+	m := &Model{
+		width:   20,
+		height:  15,
+		loading: true,
+	}
+	return m
+}
+
+func generateTable(images map[string]crt.BasicImageInfo) table.Table {
+	var rows [][]string
+	for k, v := range images {
+		imageRow := []string{k, dockerutil.CleanImageID(v.ID)[:12], humanize.Time(time.Unix(v.Created, 0)), humanize.Bytes(uint64(v.Size))}
+		rows = append(rows, imageRow)
+	}
+
+	t := table.New().
+		Border(lipgloss.NormalBorder()).
+		BorderStyle(BorderStyle).
+		StyleFunc(func(row, col int) lipgloss.Style {
+			var style lipgloss.Style
+
+			switch {
+			case row == 0:
+				return HeaderStyle
+			case row%2 == 0:
+				style = EvenRowStyle
+			default:
+				style = OddRowStyle
+			}
+
+			return style
+		}).
+		Headers("Name", "ID", "Created", "Size").
+		Rows(rows...)
+
+	return *t
+}
+
+// InitialModel returns the initial state of the model.
+func InitialModel(images map[string]crt.BasicImageInfo, standalone bool) *Model {
```

Comment:
Here as well... `InitialTUI` will make it easier to understand what it's for

Reply:
Noted and addressed.